### PR TITLE
add bootstrap-calendar@0.2.5 w/ git auto-update

### LINKED ISSUE #10241: [Request] bootstrap-calendar
**Library name:** bootstrap-calendar
**Git repository url:** https://github.com/Serhioromano/bootstrap-calendar
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

It's well supported with more than 2300 stars on github

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sufuf3**
Pull request for issue: #10241 
Related issue(s): # #
@kennynaoh please help me review this PR, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Serhioromano/bootstrap-calendar
 * Official website (optional, not the repository): http://bootstrap-calendar.azurewebsites.net/
 * NPM package url (optional): https://www.npmjs.com/package/bootstrap-calendar
 * License and its reference: BSD
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 214
   - Count of stars: 2359
   - Count of forks: 1221
 * NPM download stats (optional):
   - Downloads in the last day: 1
   - Downloads in the last week: 25
   - Downloads in the last month: 125

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/bootstrap-calendar/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "type": "git",
+    "url": "git+https://github.com/Serhioromano/bootstrap-calendar.git"
+  },
+  "keywords": [
+    "calendar"
+  ],
+  "author": {
+    "name": "Sergey Romanov"
+  },
+  "license": "BSD",
```
Comment (pvnr0082t) [inline ajax/libs/bootstrap-calendar/package.json]: Please check the license. The repo show `MIT` but `BSD` here.

File: ajax/libs/bootstrap-calendar/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "homepage": "http://bootstrap-calendar.azurewebsites.net/",
```
Comment (pvnr0082t) [inline ajax/libs/bootstrap-calendar/package.json]: @sufuf3 I can't access this url, can you? We should remove this url if it is broken.

**Comment by sufuf3** [comment]
@pvnr0082t I modified it.
Please help me review this PR again, thank you.

File: ajax/libs/bootstrap-calendar/package.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  ],
+  "author": "Sergey Romanov",
+  "license": "MIT",
+  "filename": "js/calendar.min.js",
+  "npmName": "bootstrap-calendar",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "**/calendar.*+(css|js)"
```
Comment (pvnr0082t) [inline ajax/libs/bootstrap-calendar/package.json]: It's a little bit not safe or no sense to use `**` under a non-dist folder.

File: ajax/libs/bootstrap-calendar/package.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    "type": "git",
+    "url": "git+https://github.com/Serhioromano/bootstrap-calendar.git"
+  },
+  "keywords": [
+    "calendar"
+  ],
+  "author": "Sergey Romanov",
+  "license": "MIT",
+  "filename": "js/calendar.min.js",
+  "npmName": "bootstrap-calendar",
```
Comment (pvnr0082t) [inline ajax/libs/bootstrap-calendar/package.json]: There are 9 releases on GitHub repo but 1 on NPM. Please use git auto-update config instead. Thanks.

File: ajax/libs/bootstrap-calendar/0.2.4/css/calendar.css [hunk 5]
```diff
@@ -0,0 +1,515 @@
...
+  background-color: #ededed;
+  text-align: center;
+}
+#cal-slide-box {
+  border-top: 0px solid #8c8c8c;
+}
+#cal-slide-content {
+  padding: 20px;
+  color: #ffffff;
+  background-image: url("../img/dark_wood.png");
```
Comment (pvnr0082t) [inline ajax/libs/bootstrap-calendar/0.2.4/css/calendar.css]: We need files under `img` folder also.

**Comment by sufuf3** [comment]
@pvnr0082t I modified it.
Please help me review this PR again, thank you.

**Comment by sufuf3** [comment]
@pvnr0082t  I added js/language folder, please help me review this PR again, thank you.

File: ajax/libs/bootstrap-calendar/package.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Serhioromano/bootstrap-calendar.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "js/calendar*",
+          "js/language/*",
+          "css/*",
```
Comment (pvnr0082t) [inline ajax/libs/bootstrap-calendar/package.json]: [Line24](https://github.com/cdnjs/cdnjs/pull/10445/files#diff-0e9a5cd2f24939ff230bb01b2a499c44R) in `package.json` can combine with [line22](https://github.com/cdnjs/cdnjs/pull/10445/files#diff-0e9a5cd2f24939ff230bb01b2a499c44R22).

File: ajax/libs/bootstrap-calendar/package.json [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+          "img/*"
```
Comment (pvnr0082t) [inline ajax/libs/bootstrap-calendar/package.json]: We can use `"img/**/*"` here.

**Comment by sufuf3** [comment]
@pvnr0082t I modified it.
Please help me review this PR again, thank you.

File: ajax/libs/bootstrap-calendar/package.json [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+  "license": "MIT",
+  "filename": "js/calendar.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Serhioromano/bootstrap-calendar.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "**/calendar*+(js|css)",
```
Comment (pvnr0082t) [inline ajax/libs/bootstrap-calendar/package.json]: Let's use `"+(js|css)/calendar*"` instead.

**Comment by sufuf3** [comment]
@pvnr0082t thanks for your suggest, I modified it.
Please help me review this PR again, thank you.

**Comment by sufuf3** [comment]
@pvnr0082t  I use close #8769 in commit message.
Please help me review this PR again, thank you.

**Comment by sufuf3** [comment]
@pvnr0082t I modified it.
Please help me review this PR again, thank you.

**Comment by PeterDaveHello** [comment]
For next time, I'll prefer to close duplicated issue first.

**Comment by PeterDaveHello** [comment]
Not that the PR title is wrong ... don't forget to update it.

**Comment by PeterDaveHello** [comment]
Your file permission is wrong, looks like it's from upstream, see 8387008e8e4

**Pull request merged**